integration_test/crawl_integration_test.go

```diff
@@ -37,14 +37,14 @@ func TestBadVersions(t *testing.T) {
 	}{
 		{name: "single bad version", directory: "../examples/single_bad_version", count: 1, finding: crawl.JarName | crawl.ClassPackageAndName | crawl.ClassFileMd5},
 		{name: "multiple bad versions", directory: "../examples/multiple_bad_versions", count: 13, finding: crawl.JarName | crawl.ClassPackageAndName | crawl.ClassFileMd5},
-		{name: "inside a dist", directory: "../examples/inside_a_dist", count: 2, finding: crawl.JarNameInsideArchive},
-		{name: "inside a par", directory: "../examples/inside_a_par", count: 1, finding: crawl.JarNameInsideArchive},
+		{name: "inside a dist", directory: "../examples/inside_a_dist", count: 2, finding: crawl.JarNameInsideArchive | crawl.ClassPackageAndName | crawl.ClassFileMd5},
+		{name: "inside a par", directory: "../examples/inside_a_par", count: 1, finding: crawl.JarNameInsideArchive | crawl.ClassPackageAndName | crawl.ClassFileMd5},
```

Comment:
I think you can also add the archived_fat_jar example - getting us to all the examples checked in at this point.

Reply:
We currently don't recurse into zips that are contained within an archive.
I will add this as a follow up, I'd prefer not to grow this PR anymore.

Reply:
Turns out this is still not enough, but will come later.